src/components/ZclDomainClusterView.vue

```diff
@@ -348,19 +351,44 @@ export default {
       if (hasServer) tmp.push('server')
       return tmp
     },
-    isAnyRequiredClusterNotEnabled() {
-      let lackingRequiredCluster = false
+    /* A cluster has a warning if:
+       1. It is required but disabled, OR
+       2. It is provisional and enabled */
+    doesAnyClusterHaveWarning() {
```

Comment:
isAnyClusterNotCompliant  or areAllClustersCompliant(if statements with no negations are better, easy to read)?